features/support/capybara.rb

```diff
@@ -49,6 +49,7 @@
 
   config.block_unknown_urls
   config.raise_javascript_errors = true # we would like this to be true, but need to fix our failing tests
+  config.debug = true
```

Comment:
Why?

Reply:
hahahahaha it was a temporary commit for the cucumber issue on CircleCI 😂 because it showed up first in this PR 😂